src/libraries/System.Text.Json/Common/JsonConstants.cs

```diff
@@ -6,6 +6,7 @@ namespace System.Text.Json
     internal static partial class JsonConstants
     {
         // The maximum number of parameters a constructor can have where it can be supported by the serializer.
+        public const int MaxParameterCount = 256;
         public const int MaxParameterCount = 64;
```

Comment:
This won't compile ;-)

Reply:
I went ahead and corrected the PR in case we plan to merge it before snap :-)

Reply:
:( `CTRL+Z`

(Playing around with the web-based version of vscode and codespaces... got carried away with it's awesomeness)